### DISCUSSION TITLE
Documentation about Maven central packages

### DISCUSSION THREAD

**Discussion by mahesh-hegde** [eclipse-jdt/eclipse.jdt.core#1270] [Q&A] [answered]
I (and guess many other people) use maven for most projects.

I found these on Maven central but cannot determine if they're officially endorsed:

https://central.sonatype.com/artifact/org.eclipse.jdt/org.eclipse.jdt.core/3.34.0
https://central.sonatype.com/artifact/org.eclipse.jdt/ecj/3.34.0

Also what's the difference between ECJ and JDT core? Which should I use if my purpose is just parsing Java code outside of Eclipse, from a standalone tool?

Thanks for any pointers.

**Comment by iloveeclipse** [comment, marked as answer]
> cannot determine if they're officially endorsed

Not sure what do you expect to see for "officially endorsed" proof, but they are coming from Eclipse.org, so that should be official enough?

> Also what's the difference between ECJ and JDT core? 

- ECJ is a standalone Java compiler library
- JDT core is an Eclipse bundle that provides core Java support for IDE

**Comment by mahesh-hegde** [comment]
> Not sure what do you expect to see for "officially endorsed" proof.

Thank you.

I expected a link to maven / XML snippet from official pages or github, which I didn't manage to find.
